src/le.py

```diff
@@ -2472,18 +2585,47 @@ def set_datahub_settings(self, value, should_die=True):
                     values[1])
         self.datahub = value
 
+    def validate_pathname(self, args):
```

Comment:
The validate_patchname method name is too generic. Can we use a more descriptive name?